pages/common/webpack.md

```diff
@@ -7,11 +7,11 @@
 
 `webpack {{app.js}} {{bundle.js}}`
 
-- Load css files too from the js file (this uses the css loader for `.css` files):
+- Load CSS files too from the JavaScript file (this uses the CSS loader for `.css` files):
 
 `webpack {{app.js}} {{bundle.js}} --module-bind 'css=css'`
```

Comment:
```suggestion
`webpack {{app.js}} {{bundle.js}} --module-bind '{{css}}={{css}}'`
```

Reply:
Note for me I always see the following output when I attempt to use the argument locally on both version 4 and 5. 🤔 

```bash
[webpack-cli] Error: Unknown option '--module-bind'
[webpack-cli] Run 'webpack --help' to see available commands and options
```

It looks to me that this option may have been removed in Webpack 5? But I'm not able to confirm this.

On Webpack v4 it's still listed:
https://v4.webpack.js.org/api/cli/#module-options

On Webpack v5 it's gone:
https://webpack.js.org/api/cli/

Regardless, assuming it's still in version 5:
I think it should wrap the full value, it seems based on this tutorial it's not always going to be `{{...}}={{...}}`.

> ```js
> webpack --module-bind pug-loader --module-bind 'css=style-loader!css-loader'
> ```
> \- https://timhieuvewebpack.blogspot.com/#cli

```suggestion
`webpack {{app.js}} {{bundle.js}} --module-bind '{{css=css}}'`
```

If it was indeed removes, it could be nice to remove from here to make space for more relevant commands.